classes/services/PKPSchemaService.php

```diff
@@ -53,6 +53,8 @@ class PKPSchemaService
     public const SCHEMA_USER_GROUP = 'userGroup';
     public const SCHEMA_EVENT_LOG = 'eventLog';
     public const SCHEMA_EMAIL_LOG = 'emailLog';
+    public const SCHEMA_GENRE = 'genre';
```

Comment:
I don't think we need to use schema. Take a look at the `ModelWithSettings` trait, it should have everything required. I believe @touhidurabir has some examples